.github/workflows/payloads/pr_payload.json

```diff
@@ -0,0 +1,12 @@
+{
+    "blocks": [
+
+        {
+            "type": "section",
+            "text": {
+                "type": "mrkdwn",
+                "text": "G3 CI/PR-${{ env.pr_number }}: ${{ env.current_job_status }}\n```Time   : ${{ env.overall_time }}\nBranch : ${{ env.branch }}\nPR     : ${{ env.pr_url }}\nDiff   : ${{ env.diff}}\n\n    ${{ env.commit_message }}```"
```

Comment:
Shall we remove the whitespaces before the **${{ env.commit_message }}** in the last line?

Reply:
This was modeled after other examples, using spaces as a separator. We can omit them, and use a "Message: " prefix. No opinion on this one, as it is a visual change.

Reply:
In jenkins we use the `git log -n 1` output and we use it as is replacing the `commit` with  `Commit:` prefix [[link1](https://github.com/Workable/qa-ci/blob/master/vars/git.groovy#L9-L17), [link2](https://github.com/Workable/qa-ci/blob/master/vars/reporting.groovy#L11)] so that we print both the commit hash, author, date and message.

[Example:](https://workable.slack.com/archives/C7KSZ7831/p1695887174545409)
<img width="407" alt="Screenshot 2023-09-28 at 10 52 23" src="https://github.com/Workable/g3/assets/42936184/fe31db0a-0d4e-4d5d-af7a-3d7f2ac7ca92"> 

If this is not possible I suggest we simply print the commit message without any whitespace in front.

Reply:
All of that information is available. Will add it in the next commit. 📝

Reply:
All requested information should now be present.